### ISSUE TITLE
Scale slider to match main column width

### ISSUE THREAD

**Issue by pvanheus** [pvanheus/1976_map_vue#7]
On Firefox the slider width does not match the main column width:

![image](https://user-images.githubusercontent.com/4154788/61702034-6923a780-ad3f-11e9-92b6-eed53c2793fb.png)

**Comment by clivecorbishley** [comment]
I took a look at this one - and while I don't have an answer, I did find some quite weird behaviour around this bug.

I tried to open developer tools to tweak the styling when I noticed that it fixed itself as the developer tools open. If you then reload the page, it breaks again. If you then proceed to close the still open developer tools, it fixes itself again. I've attached a gif below to show what I saw.

![2019-07-23 22-45-18 2019-07-23 22_48_00](https://user-images.githubusercontent.com/8817785/61746477-0ca2a580-ad9c-11e9-92f7-a7e288d548b3.gif)

I wouldn't rule out the possibility of this being a bug with Firefox

**Comment by Baadier-Sydow** [comment]
@clivecorbishley I picked up the same behaviour when I had a look. Possibly linked to events that trigger or an update after change detection?

**Comment by clivecorbishley** [comment]
@Baadier-Sydow Yeah - that makes sense! I'm thinking the best approach to this would be to report the bug to firefox (I'd be happy to take this on), but still try fix it up with some css or html on this side as well just in case I'm wrong or Firefox takes a while. Any thoughts?

**Comment by Baadier-Sydow** [comment]
@clivecorbishley Mind if I have a look at it before you report to Firefox? You know how it goes, each reported bug has a cost :) So if possible, I want to check if its something we can resolve first before reporting.

**Comment by clivecorbishley** [comment]
@Baadier-Sydow For sure 👍 Go for it :)

**Comment by Baadier-Sydow** [comment]
Bizarre. It seems to be working on my end now? Perhaps linked to Firefox version as I don't see any new commits on the repo.

<img width="1392" alt="Screen Shot 2019-08-01 at 9 17 01 PM" src="https://user-images.githubusercontent.com/5681635/62321403-d69fa880-b4a2-11e9-9e6a-dd97e1db4430.png">

<img width="753" alt="Screen Shot 2019-08-01 at 9 16 52 PM" src="https://user-images.githubusercontent.com/5681635/62321385-c8518c80-b4a2-11e9-998d-254c5e4027be.png">

@clivecorbishley can you check on your end?

**Comment by pvanheus** [comment]
Not working with 68.0.1 (on Ubuntu 19.04), but I see you have 69.0b9, which is scheduled for release in September.

**Comment by Baadier-Sydow** [comment]
@pvanheus Let me download 68 to see if I can reproduce on my end. I'm on a MacBook.

**Comment by Baadier-Sydow** [comment]
@pvanheus @clivecorbishley that probably makes it appear like a browser bug but let me check first.

**Comment by Baadier-Sydow** [comment]
> Not working with 68.0.1 (on Ubuntu 19.04), but I see you have 69.0b9, which is scheduled for release in September.

Its reproducible on my end on 68.01
